api/tests/Traits/CrawlerTrait.php

```diff
@@ -0,0 +1,732 @@
+<?php
+
+use Illuminate\Http\Request;
+use Symfony\Component\DomCrawler\Form;
+use Symfony\Component\DomCrawler\Crawler;
+use PHPUnit_Framework_ExpectationFailedException as PHPUnitException;
+
+/**
+ * With the future release of Lumen 5.1, this trait can most probably be
```

Comment:
Can you add an `@todo` into this comment so that we can find it again later (PHPStorm convention for marking todo items)